sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
     <Nullable>enable</Nullable>
     <DefineConstants>$(DefineConstants);AZURE_NULLABLE;HAS_INTERNALS_VISIBLE_CORE</DefineConstants>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net462;net472;net6.0</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net462;net472;net8.0</TargetFrameworks>
```

Comment:
RequiredTargetFrameworks includes net8.0 unless there is an override. So this net6.0 target can just be removed.